invenio_rdm_records/theme/templates/invenio_rdm_records/details/list_entities.html

```diff
@@ -8,16 +8,19 @@
 
 {% macro list_entities(entities) %}
   {% for entity in entities %}
-    {# TODO: Is the first identifier given precedence or is "Orcid" given precedence? #}
-    {# Assume first identifier is given precedence #}
-    {% set main_identifier = entity.identifiers[0] if entity.identifiers and entity.identifiers[0] else {"scheme": ""} %}
-    {%- if main_identifier.scheme == "Orcid" %}
-    <a href="{{ main_identifier.identifier|pid_url('orcid') }}">
-      <img class="inline-orcid" src="{{ url_for('static', filename='images/orcid.svg') }}" />
-    </a>
-    {# TODO: Other potential main identifiers #}
+    {# TODO: Revisit when more than ORCiD accepted #}
```

Comment:
I think this todo can be removed? We are accepting orcid and ror now?

Reply:
We might accept other identifiers (Zenodo does like GND...) in the future, but we haven't had a discussion about it too much, so we can leave the TODO there for now.